src/core/annotation.js

```diff
@@ -2681,11 +2681,18 @@ class TextWidgetAnnotation extends WidgetAnnotation {
   constructor(params) {
     super(params);
 
+    const { dict } = params;
+
+    if (dict.has("PMD")) {
+      // It's used to display a barcode but it isn't specified so we just hide
+      // it to avoid any confusion.
+      this.flags |= AnnotationFlag.HIDDEN;
+      this.data.hidden = true;
```

Comment:
Similar to e.g. https://github.com/mozilla/pdf.js/blob/master/src/core/annotation.js#L3325, let's add a `warn("Barcodes are not supported");` to aid future debugging if users report bugs about missing elements so we can easily spot this in the web console.

Reply:
Oh yes I forgot, thx.